Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM node:10.17.0-buster-slim
```

Comment:
Should likely not be a node flavor of Linux @AndrewHUNGNguyen - probably doesn't matter now, but it could reduce the complexity and size of the docker container

Reply:
Ahh ok sounds good, I just thought to keep this version of node since this was the "working" node version that maintains the working behavior of node.